Sources/BraintreePayPal/BTPayPalVaultEditResult.swift

```diff
@@ -1,8 +1,33 @@
 import Foundation
+import BraintreeCore
```

Comment:
We need to use the `canImport` syntax here to resolve the cocoapods error

Reply:
Oh, nice catch.